### PR TITLE
Add doc for chef-server 14.4

### PR DESCRIPTION
**Pull request by tas50**
It's out!

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/release_notes_server.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ Chef Infra Server acts as a hub for configuration data by storing
...
+
+PostgreSQL was upgraded from 9.6.21 to 9.6.22 to resolve [CVE-2021-22137](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32027) and [CVE-2021-22137](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32028).
+
+#### Elasticsearch
+
+Elasticsearch was upgraded from 6.8.14 to 6.8.15 to resolve [CVE-2021-22137](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22137).
+
+#### OpenResty Version in Errors
+
+Error pages will no longer display the running version of OpenResty.
```
Comment (kagarmoe) [inline content/release_notes_server.md]:
```suggestion
Error pages no longer display the running version of OpenResty.
```

File: content/release_notes_server.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ Chef Infra Server acts as a hub for configuration data by storing
...
 
+## What's New in 14.4
+
+### Security Updates
+
+#### PostgreSQL
+
+PostgreSQL was upgraded from 9.6.21 to 9.6.22 to resolve [CVE-2021-22137](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32027) and [CVE-2021-22137](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32028).
+
+#### Elasticsearch
```
Comment (kagarmoe) [inline content/release_notes_server.md]:
```suggestion
#### Elasticsearch 6.8.15
```

File: content/release_notes_server.md [hunk 3]
```diff
@@ -16,6 +16,22 @@ Chef Infra Server acts as a hub for configuration data by storing
 cookbooks, the policies that are applied to nodes, and metadata that
 describes each registered node that is managed by the Chef Infra Client.
 
+## What's New in 14.4
+
+### Security Updates
+
+#### PostgreSQL
```
Comment (kagarmoe) [inline content/release_notes_server.md]:
```suggestion
#### PostgreSQL 9.6.22
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Added information to the headers to make searching for known releases simpler.
Changed a future tense to present tense.
We need to come up with a formulaic statement for upgrades that doesn't use the simple past passive.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**